spec/lib/services/nodes-api-service-spec.js

```diff
@@ -736,8 +737,16 @@ describe("Http.Services.Api.Nodes", function () {
     });
 
     describe('Tagging', function() {
+
         var node = {
-            id: '1234abcd1234abcd1234abce'
+            id: '1234abcd1234abcd1234abcd',
+            name: 'name1',
+            type: 'compute',
+        };
+        var node1 = {
+            id: '5678efgh5678efgh5678efgh',
+            name: 'name1',
```

Comment:
findByTags is stubbed such that the tag is not actually needed, but for what its worth, the tags are in the 'tags' property, but the test references the value of the 'name' property, which may be confusing.  Can you change this to be "tags: [ 'name1' ]  " ?